A job failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/19230465231/job/54967645254) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/54967645254) [(plain)](https://github.com/rust-lang/rust/commit/581399837e47ff3ee71a4580e358ef9187c10d52/checks/54967645254/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  PR_CI_JOB: 1
  SKIP_SUBMODULES: src/gcc
  IMAGE: tidy
##[endgroup]
error: PRs changing the `nightly` channel should be sent to the `master` branch!
##[error]Process completed with exit code 1.
##[group]Run echo "disk usage:"
echo "disk usage:"
df -h
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
```

</details>

--- a/src/librustdoc/passes/collect_intra_doc_links.rs
+++ b/src/librustdoc/passes/collect_intra_doc_links.rs
@@ -1215,7 +1215,6 @@ impl LinkCollector<'_, '_> {
             || !did.is_local()
     }
 
-    #[allow(rustc::potential_query_instability)]
     pub(crate) fn resolve_ambiguities(&mut self) {
         let mut ambiguous_links = mem::take(&mut self.ambiguous_links);
         for ((item_id, path_str), info_items) in ambiguous_links.iter_mut() {

